src/cat-data-service/src/service/common/objects/voting_key.rs

```diff
@@ -0,0 +1,12 @@
+use poem_openapi::{types::Example, NewType};
+use serde::Deserialize;
+
+#[derive(NewType, Deserialize)]
+#[oai(example = true)]
+pub(crate) struct VotingKey(pub String);
```

Comment:
Also rename the file to "voting_public_key" so its clear it the public key and not the private key.
```suggestion
pub(crate) struct VotingPublicKey(pub String);
```